config.js

```diff
@@ -968,7 +968,7 @@ var config = {
         // The interval at which rtcstats will poll getStats, defaults to 1000ms.
         // If the value is set to 0 getStats won't be polled and the rtcstats client
         // will only send data related to RTCPeerConnection events.
-        // rtcstatsPolIInterval: 1000,
+        // rtcstatsPollInterval: 1000,
```

Comment:
Maybe update the default value here as well?

Reply:
Argh. Sorry. GitHub website showed me everything green. So I pressed the merge button. And only after the merge it showed me your review comment. I should have reloaded the page before merging.